php/router.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// Used by `wp server` to route requests.
+
+namespace WP_CLI\Router;
+
+function get_full_host( $url ) {
```

Comment:
Because it's used twice, to compare homeurl and siteurl. It's probably not the best name.